### ISSUE TITLE
Change Scene Functionality to declare named objects

### ISSUE THREAD

**Issue by ChamzasKonstantinos** [KavrakiLab/vicon_object_server#32]
Currently when a scene is loaded all the objects must be different yaml files each one specifying the geometry of the object and the offset. However in many cases many objects share their geometry e.g a bookshelf has the same boxes in different places, and most of the time we have multiple cubes or cylinders of the same geometry. Instead of having can1.yaml can2.yaml we should only have one file named can.yaml that has the geometry and the rest should be specified in the scene.yaml. It is extremely bothersome to have to change 10 files when you want to change the geometry of 1 can.  I am opening this to discussion and ideas on implementation.

**Comment by zkingston** [comment]
I agree that something like this should be how objects are defined. However, I'd say that it isn't worth it to redo the implementation of the current server. We should instead focus on the development of the next version of the server, where we can make some of the more sweeping changes that have been discussed offline.
